### ISSUE TITLE
ServiceModel SwaggerUI Error

### ISSUE THREAD

**Issue by EricKwon94** [max-ieremenko/ServiceModel.Grpc#301]
**Describe the bug**
When using `UseServiceModelGrpcSwaggerGateway` together with `AuthorizeAttribute`, the following error appears in Swagger UI:
```
System.ArgumentOutOfRangeException: Specified argument was out of the range of valid values. (Parameter 'start')
   at System.ThrowHelper.ThrowArgumentValidationException(Array array, Int32 start)
   at System.Buffers.ReadOnlySequence`1..ctor(T[] array, Int32 start, Int32 length)
   at ServiceModel.Grpc.AspNetCore.Internal.Swagger.SwaggerUiRequestHandler.WriteResponseMessageAsync(MemoryStream original, PipeWriter bodyWriter, IMarshallerFactory marshallerFactory, IOperationDescriptor descriptor, CancellationToken token)
   at ServiceModel.Grpc.AspNetCore.Internal.Swagger.SwaggerUiMiddleware.HandleRequestAsync(HttpContext context, IMarshallerFactory marshallerFactory, IOperationDescriptor descriptor)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)
```
However, the client (e.g., via console) functions correctly and returns a 401 Unauthorized as expected.

**To Reproduce**
Please provide specific steps to reproduce the bug.

**Expected behavior**

**Environment**
 - OS: [e.g. Windows/Linux and version]
 - .net framework version

**Additional context**
Here is sample code.
[GrpcBareBones.zip](https://github.com/user-attachments/files/17955726/GrpcBareBones.zip)

**Comment by max-ieremenko** [comment]
Hi @EricKwon94,

thank you for detailed bug description. The `SwaggerUiMiddleware` does not take into account `Response.StatusCode 401`.
It will be fixed.

In the meantime, you can change the middleware stack a little bit to set authorization first

```c#
app.UseHttpsRedirection();
app.UseAuthentication();
app.UseAuthorization();

// Configure the HTTP request pipeline.
if (app.Environment.IsDevelopment())
{
	app.UseSwagger();
	app.UseSwaggerUI();
	app.UseServiceModelGrpcSwaggerGateway();
}

// app.UseHttpsRedirection();
// app.UseAuthentication();
// app.UseAuthorization();

app.MapGrpcService<SimpleServiceModel>();
```

in this case the behavior will be identical to your client.

> the client (e.g., via console) functions correctly and returns a 401

It works because `app.Environment.IsDevelopment() is false` and authentication stops process request.

**Comment by max-ieremenko** [comment]
the fix is released in [version 1.11.1](https://github.com/max-ieremenko/ServiceModel.Grpc/releases/tag/1.11.1)
